### DISCUSSION TITLE
Automaticaly open the sidebar mod panel on posts

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#16558] [Feature Requests]
For a moderation session, make the Mod panel with the buttons for upvote/dowvote and flag in the sidebar open by default if an option is enabled. 
What would that do for us as a mod? 

To waste less time opening the sidebar, especially when there is a raid of posts that we must quickly flag. 

___

I created a JS script to automagically ✨ open the sidebar. If any would like to participate to the script : 

https://github.com/devtotools/open-devto-panel-sidebar-mod

PS : I don't know how to make it so that when we click on "Click here to install the script." ([link](https://github.com/devtotools/open-devto-panel-sidebar-mod#already-have-tampermonkey-or-alternative-)) it sends us to the local page of TamperMonkey to install the script.

**Comment by msarit** [comment]
In my mind, the Mod Center (`/mod`) lends itself nicely for moderating multiple articles on one page.

**Reply by thomasbnt** [reply]
Yeah of course, is pretty cool panel. But to see the post in full context without distraction, maybe better.
